packages/dashboard/src/components/side-panel/sub-components/thresholds-section.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ExpandableSection } from '@cloudscape-design/components';
+import ExpandableSectionHeader from '../shared/expandableSectionHeader';
+
+const ThresholdsSection = () => {
+  return (
+    <ExpandableSection
+      headerText={<ExpandableSectionHeader text="Thresholds" onClickButton={() => {}} />}
```

Comment:
These are supposed to be ExpandableSections, can we add that functionality in this same pull request?

Reply:
Same - ExpandableSection handles that functionality.